src/main.rs

```diff
@@ -33,38 +33,55 @@ pub fn main() -> Result<()> {
 
     let (args, state_file) = cli::init_cli();
 
-    // setup platform (in our case, PC)
-    let trussed_platform = platform::init_platform(state_file);
-
-    // setup Trussed
-    let mut trussed_service = trussed::service::Service::new(trussed_platform);
-    let client_id = "totp";
-    // In real life, `trussed_service.try_new_client` has an additional parameter that is a `Syscall`
-    // implementation; giving the client a way to signal the ambient runtime to call the service.
-    // Here, we use the service's implementation of `Syscall`, where it simply calls itself :)
-    let trussed_client = trussed_service.try_as_new_client(client_id).unwrap();
-
-    // setup authenticator
-    let mut authenticator = authenticator::Authenticator::new(trussed_client);
-
+    //setup wireguard
+    let trussed_platform_wg = platform::init_platform(state_file.clone());
+    let mut trussed_service_wg = trussed::service::Service::new(trussed_platform_wg);
+    let client_id_wg = "wireguard";
```

Comment:
Could use `wg` here, as this client ID goes in all filenames on the flash. Premature optimization probably xD